doc/other_modules/jenny/language/commands/commands.md

```diff
@@ -0,0 +1,16 @@
+# Commands
+
+The **commands** are special instructions surrounded with double angle-brackets: `<<stop>>`. There
```

Comment:
isn't this part of the jenny spec that we can link to? or is this specific to our port?

Reply:
There are still some parts of the documentation missing, I'm planning to add them in later PRs, since this one became too big already.

YarnSpinner docs have a section about commands (https://docs.yarnspinner.dev/getting-started/writing-in-yarn/commands), but for some reason they only document 2 of them.